package.json

```diff
@@ -3,7 +3,7 @@
    "displayName": "Developer Tools for Azure Kubernetes Service (AKS)",
    "description": "Create deployment files and configure GitHub Actions workflows to deploy applications to Azure Kubernetes Service (AKS).",
    "publisher": "ms-kubernetes-tools",
-   "version": "0.1.1",
```

Comment:
💡Please note:  There is no need to make this version change as part of this PR, this becomes the part of release PR aka - `publish PR ` where cahngelog file changes and package.json changes go hand in hand. Thanks.

Reply:
Nice catch, ill revert!